vhdbuilder/packer/components.json

```diff
@@ -2,351 +2,425 @@
   "ContainerImages": [
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/kubernetes-dashboard:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.10.1"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v2.0.1"
+      ],
+      "multiArchVersions": [
+        "v2.4.0"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-scraper:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.0.4"
+      ],
+      "multiArchVersions": [
+        "v1.0.7"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/exechealthz:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.2_v0.0.5"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.8.5",
         "1.7"
+      ],
+      "multiArchVersions": [
```

Comment:
When updating components in the future, how can we determine if/when to update the `amd64OnlyVersions` or `multiArchVersions` or both?

Reply:
I have a wiki to track arm64 porting status of AKS addon container images owned by different owners. Basically if a container image tag can be used on amd64 only, it should be added into amd64OnlyVersions. If an image tag has 'multi-arch'(can be used by both amd64/arm64), it should be added into multiArchVersions.
https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/182948/Addons-binaries-and-images-on-Agent-Nodes